### ISSUE TITLE
[USABILITY FIX] KYC - change flow logics and wording for last steps

### ISSUE THREAD

**Issue by glupiawymowka** [Neufund/platform-frontend#1934]
At the moment, the user doesn't have any clear popup informing that he successfully submitted the required data.

1. last step to go - the user is uploading documents. When he did it, he should see the "SUBMIT" button instead of "Continue" so he knows what's going on - **please change translation for the button**

![image.png](https://images.zenhubusercontent.com/5ae1e2334b5806bc2bd0c3bb/1213a6e5-561f-47ed-bfad-55c8202f216b)

2. Next step is : user is confirming his identity verification request. After he confirms, he should receive a popup (we already have it, at the point where user finished his KYC and chooses to submit additional documents; **it's wrong, please show this popup exactly when user clicks to **confirm** his KYC**)

the popup we have: 
![image.png](https://images.zenhubusercontent.com/5ae1e2334b5806bc2bd0c3bb/4195dd3e-8d4f-4e04-baae-18668889bb9b)

**please add new wording for the popup**

header : “Thank you for submitting your identity verification request!”

paragraph : “We will review your documents within 1 business day and contact you if we require further information. Otherwise, we will notify you by email once your account has been successfully verified."

3. Clicking OK on this popup **should redirect back to profile**

4. User is back in his profile. Only now he can decide to upload additional documents. If he chooses so, **please redirect him to this page:**

![image.png](https://images.zenhubusercontent.com/5ae1e2334b5806bc2bd0c3bb/3fda80b8-afaa-4229-9353-7e194c254e95)

TO SUM UP : 

* user clicks SUBMIT button to send data to backoffice
* he confirms the request
* popup with new wording appears
* OK redirects him to the profile
* from KYC widget, user can go to "add more documents" screen (no popup inbetween)
![image.png](https://images.zenhubusercontent.com/5ae1e2334b5806bc2bd0c3bb/1eb96b42-123e-413e-93e2-8d8f9832af07)

**Comment by glupiawymowka** [comment]
accidentally checked also this one :D 
I guess it's done
![image](https://user-images.githubusercontent.com/18315589/48916370-55f3cb80-ee82-11e8-8d1e-68954ba934b5.png)
